test/valgrind/appendContextElement.vtest

```diff
@@ -1,31 +0,0 @@
-# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
All .vtest files removed? I understand the request they are using are over-covered in .test, right?

Reply:
Yes, we cover many times over already what's in the old vtest files. Haven't used them in a few years. It was time to remove them

Reply:
NTC